### PR TITLE
Change rem to px for comment margin

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
changes rem unit to px

## Why?
unit clarity and consistency

### CODE REVIEW

File: src/components/Comment/Comment.tsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ const commentCss = css`
   display: block;
   clear: left;
   ${textSans.small()}
-  margin-top: 0.375rem;
-  margin-bottom: 0.5rem;
+  margin-top: 6px;
```
Comment (oliverlloyd) [inline src/components/Comment/Comment.tsx:59]: `space[1]`?

File: src/components/Comment/Comment.tsx [hunk 2]
```diff
@@ -56,8 +56,8 @@ const commentCss = css`
   display: block;
   clear: left;
   ${textSans.small()}
-  margin-top: 0.375rem;
-  margin-bottom: 0.5rem;
+  margin-top: ${space[2]}px;
+  margin-bottom: ${space[3]}px;
```
Comment (SiAdcock) [inline src/components/Comment/Comment.tsx:60]: [Best of both worlds](https://github.com/guardian/source/blob/master/src/core/foundations/src/space.ts#L19):

```suggestion
  margin-top: ${remSpace[2]}px;
  margin-bottom: ${remSpace[3]}px;
```

Comment (liywjl) [reply src/components/Comment/Comment.tsx:60]: nice! I want to know how you did a multi-line suggestion?

Comment (SiAdcock) [reply src/components/Comment/Comment.tsx]: It's not super-obvious:

1. Click and drag the line numbers to highlight multiple lines
2. Click the blue ➕ icon next to the bottom line number
3. Click "insert a suggestion"

**Pull request merged**